app/views/root/completed_transaction.html.erb

```diff
@@ -38,47 +38,9 @@
     </div>
   <% end %>
 
-  <h2 class="satisfaction-survey-heading">Satisfaction survey</h2>
-
-  <form class="contact-form" action="/contact/govuk/service-feedback" method="post" id="completed-transaction-form">
-    <input type="hidden" id="service_slug" name="service_feedback[slug]" value="<%= @publication.slug.gsub("done/", "") %>" />
-    <input type="hidden" id="service_done_page_url" name="service_feedback[url]" value="<%= @publication.web_url %>" />
-    <fieldset>
-      <legend><h2>Overall, how did you feel about the service you received today?</h2></legend>
-      <br />
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="very-satisfied" value="5">
-      <label for="very-satisfied">Very satisfied</label>
-      <br />
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="satisfied" value="4">
-      <label for="satisfied">Satisfied</label>
-      <br />
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="neither-satisfied-or-dissatisfied" value="3">
-      <label for="neither-satisfied-or-dissatisfied">Neither satisfied or dissatisfied</label>
-      <br />
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="dissatisfied" value="2">
-      <label for="dissatisfied">Dissatisfied</label>
-      <br />
-      <input type="radio" required name="service_feedback[service_satisfaction_rating]" id="very-dissatisfied" value="1">
-      <label for="very-dissatisfied">Very dissatisfied</label>
-    </fieldset>
-
-    <br />
-
-    <fieldset>
-      <legend><h2>How could we improve this service?</h2></legend>
-      <br />
-      <label for="improvement-comments" class="visuallyhidden">Your comments</label>
-      <textarea name="service_feedback[improvement_comments]" class="full-size counted" id="improvement-comments">
-      </textarea>
-       <p id="improvement-commentscounter" class="hint" aria-live="polite" aria-atomic="false">(Limit is 1200 characters)</p>
-
-      <div id="transaction-completed-form-notice">
-        <p>Please don't include any personal or financial information, for example your National Insurance or credit card numbers.</p>
-      </div>
-    </fieldset>
-    <br />
-    <p class="action group">
-      <button type="submit" class="button">Send feedback</button>
-    </p>
-  </form>
+  <% if AssistedDigitalSatisfactionSurvey.show_help_with_fees_survey? @publication.slug %>
+    <%= render 'assisted_digital_help_with_fees_satisfaction_survey', publication: @publication %>
```

Comment:
I like the use of the partial here. It's not part of our changes but do you feel like it'd be a "quick win" to also have partials for the `register_to_vote` and `organ_donor` blocks based on the publication promotion choice above?